src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/upgrade/upgrade.component.html

```diff
@@ -46,18 +46,28 @@ <h5>{{ version }}</h5>
              i18n-aria-label
              id="clusterStatus">
       <div class="d-flex flex-column justify-content-center align-items-center">
-        <ng-template #healthChecks>
-          <cd-health-checks [healthData]="healthData.health.checks"></cd-health-checks>
-        </ng-template>
+
         <ng-template #healthWarningAndError>
-        <div class="info-card-content-clickable mt-1"
+        <!-- <div > -->
+          <cds-toggletip
              [ngStyle]="healthData.health.status | healthColor"
-             [ngbPopover]="healthChecks"
+             [isOpen]="isOpen"
+             [dropShadow]="true"
+             [caret]="true"
```

Comment:
from the [documentation](https://angular.carbondesignsystem.com/?path=/docs/components-toggletip--docs), I see `true` is the default values for these inputs. So do we want to add it here? For the sake of making less code changes, we could remove it.